File: sdkwork-app-sdk-csharp/Models/NotificationSettingsUpdateForm.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class NotificationSettingsUpdateForm
    {
        public bool? EnablePush { get; set; }
        public bool? EnableEmail { get; set; }
        public bool? EnableSms { get; set; }
        public bool? EnableInApp { get; set; }
        public string? QuietHoursStart { get; set; }
        public string? QuietHoursEnd { get; set; }
        public string? NotificationSound { get; set; }
        public bool? VibrationEnabled { get; set; }
    }
}

File: sdkwork-app-sdk-typescript/src/types/notification-settings-update-form.ts
/** 通知设置更新表单 */
export interface NotificationSettingsUpdateForm {
  /** 是否启用推送通知 */
  enablePush?: boolean;
  /** 是否启用邮件通知 */
  enableEmail?: boolean;
  /** 是否启用短信通知 */
  enableSms?: boolean;
  /** 是否启用应用内通知 */
  enableInApp?: boolean;
  /** 免打扰开始时间 */
  quietHoursStart?: string;
  /** 免打扰结束时间 */
  quietHoursEnd?: string;
  /** 通知声音 */
  notificationSound?: string;
  /** 是否振动 */
  vibrationEnabled?: boolean;
}
